package.json

```diff
@@ -10,6 +10,8 @@
     "eslint-plugin-react": "^7.10.0",
     "react": "^16.4.0",
     "react-dom": "^16.4.0",
+    "react-router": "^4.3.1",
+    "react-router-dom": "^4.3.1",
```

Comment:
according to this discussion, seems right to add only react-router-dom.
https://github.com/ReactTraining/react-router/issues/4648

Reply:
good point, react-router-dom should have everything needed for web.
I will remove react-router